test/createTest.m

```diff
@@ -0,0 +1,81 @@
+%==========================================================================
+%This function writes a function, testDataSets, for testing the viewer on
+%all jsons stored locally in the Data/jsons directory. 
+%
+%Authors: Thomas Maullin, Camille Maumet, Thomas Nichols
+%==========================================================================
+
+function createTest()
+       
+    %Create new test file for editing.
+    FID = fopen(fullfile(fileparts(mfilename('fullpath')), 'testDataSets.m'),'wt');
+
+    %Add the start of the test code.
+    start = sprintf('%s' , '%%==========================================================================',...
```

Comment:
Wow, I did not realise that you were actually writing the test code within the test code! Usually this is done with "data-driven testing" that can take a parameter as input and then generates a series of tests. More info on DDT: http://defragdev.com/blog/?p=660. We can keep this as it is as I am not sure if there is an easy way to do DDT in Matlab. The only thing is that you might want to add `test/testDataSets.m` on GitHub. This will avoid having an untracked file after running the tests:
```
$ git status 
On branch FSLGroupData
Your branch is up-to-date with 'tomm/FSLGroupData'.
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	test/testDataSets.m
```